### DISCUSSION TITLE
Is there any features to hide private team in organization?

### DISCUSSION THREAD

**Discussion by rwdkor** [community/community#3594] [Discussions]
I changed my team to private, but others in the organization still can see team in list.

**Comment by rwdkor** [comment]
![1](https://user-images.githubusercontent.com/49826526/112074675-2708d400-8bba-11eb-96f0-3533eae53e83.png)

**Comment by PranamBhat** [comment]
Hi

Go to "Settings" and make these settings as per your need :


![image](https://user-images.githubusercontent.com/32120033/112141571-5543df00-8bcd-11eb-91c6-85dfb0942d92.png)
